app/src/main/java/com/android/example/camerax/tflite/CameraActivity.kt

```diff
@@ -182,6 +182,8 @@ class CameraActivity : AppCompatActivity(), GLSurfaceView.Renderer {
             Log.w("sessionUpdate", Thread.currentThread().name)
 
             surfaceView.queueEvent {
+                //session.setCameraTextureName(0)
+                session!!.setCameraTextureNames(intArrayOf(0))
```

Comment:
I think I need to use surfaceview here but I am not sure how to get texture ID for that. I don't see any similar helper function available.